openff/interchange/components/potentials.py

```diff
@@ -161,12 +172,14 @@ def get_mapping(self) -> Dict:
         return mapping
 
     def parametrize(self, p=None):
+        """Return an array of system parameters, given an array of force field parameters."""
         if p is None:
             p = self.get_force_field_parameters()
 
         return self.get_system_parameters(p=p)
 
     def parametrize_partial(self):
+        """Return parametrize() modified with functools.partial gluing on `mapping`."""
```

Comment:
I'm not sure how familiar the typical Pythonista is with functools...

```suggestion
        """Return a function that will call `self.parametrize()` with arguments specified by `self.mapping`."""
```

This may not be correct as the parametrize method doesn't have a mapping argument and I didn't think `partial` does either so I might just be confused.